ONSdigital/eq-questionnaire-runner
# List schemas under type on endpoint

### What is the context of this PR?

This PR will make changes to allow loading schemas in Runner build for the new schema folder structure and expose schemas in schemas endpoint under the correct folder/type.

Related [launcher PR](https://github.com/ONSdigital/eq-questionnaire-launcher/pull/35)

### How to review 
- Make sure schemas endpoint lists schemas under type
- `make load-schemas` works as expected
- Use launcher to load schemas and make sure schemas are listed under correct type
- Schemas can be launched in launcher

### Checklist
